pubsub/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/pubsub/PubSubSpec.scala

```diff
@@ -39,7 +39,7 @@ class PubSubSpec extends WordSpec with Matchers with BeforeAndAfterAll {
   }
   val system = app.actorSystem
   Cluster.get(system).join(Cluster.get(system).selfAddress)
-  implicit val mat = ActorMaterializer.create(system)
+  implicit val mat = Materializer.matFromSystem(system)
```

Comment:
This was not the intended usage of `matFromSystem`, which is an implicit conversion.
Instead the idea was to only have
```
implicit val system: ActorSystem
```

and then the streams can be run with that instead of having an `implicit val mat`